tests/cases/fourslash/completionsGenericIndexedAccess6.ts

```diff
@@ -0,0 +1,23 @@
+// @Filename: component.tsx
```

Comment:
Shouldn't this file have a `/// <reference path="fourslash.ts" />` at the top for completion support while editing?

Reply:
I often miss these because editor support works as long as you have another file open with the triple-slash 😦

Reply:
hah, yeah.